src/ManifestStrategy/ConfigureEnvManifestStrategy.php

```diff
@@ -90,6 +90,33 @@ public function apply(array $manifest, string $moduleName, InputOutputInterface
         return true;
     }
 
+    /**
+     * @param string $target
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     *
+     * @return bool
+     */
+    protected function targetExists(string $target, InputOutputInterface $inputOutput): bool
+    {
+        $configFileName = $this->config->getConfigPath();
+        $configFileContent = file_get_contents($configFileName);
+
+        if ($configFileContent === false) {
+            $inputOutput->writeln(sprintf(
+                'Could not read from file `%s`. Please check file path or customize using `IntegratorConfig::getConfigPath()`.',
```

Comment:
`IntegratorConfig::getConfigPath()` doesn't look like something we can customize a string with.